src/api/error.rs

```diff
@@ -32,13 +32,13 @@ pub enum Error {
 	Metadata(MetadataError),
 	InvalidMetadata(InvalidMetadataError),
 	NodeApi(ac_node_api::error::Error),
-	StorageValueDecode(codec::Error),
```

Comment:
Renamed to Codec, as it's already used for general decoding errors, not just StorageValue.